test_codecept/integration/tests/get_Organisation_User_Details.ts

```diff
@@ -9,9 +9,9 @@ suite('Manage Org -> Get Organisation User details', function() {
       response.status.should.be.eql(200);
       console.log(response.data.users[0].email);
       // response.data.users[14].userIdentifier.should.be.eql('fb25e17b-2456-4ce0-909d-071b68d10d59');
-      response.data.users[0].firstName.should.be.eql('Vamshi');
-      response.data.users[0].lastName.should.be.eql('Muniganti');
+      response.data.users[0].firstName.should.be.eql('Jason');
+      response.data.users[0].lastName.should.be.eql('Lee');
       // response.data.users[0].email.should.be.eql('xuiapitestuser@mailnesia.com');
-      response.data.users[0].idamStatus.should.be.eql('PENDING');
```

Comment:
Again - search for a specific user and then check that that user's status is as expected